app/models/submission_collector.rb

```diff
@@ -212,6 +226,7 @@ def manually_collect_submission(grouping, rev_num, async=true)
     grouping.save
 
     new_submission = Submission.create_by_revision_number(grouping, rev_num)
+    new_submission = apply_penalty_or_add_grace_credits(grouping, apply_late_penalty, new_submission)
```

Comment:
Useless assignment to variable - `new_submission`.<br>Line is too long. [101/80]